rust/src/cli.rs

```diff
@@ -260,6 +264,13 @@ fn main_stats(ws: &Workspace) -> Result<()> {
     Ok(())
 }
 
+fn main_clean(opts: &Options) -> Result<()> {
+    for path in &opts.paths {
+        clean_workspace(path)?;
```

Comment:
If this supports multiple directories, it should clean them all. Here is might clean the first one, fail on the second argument, and stop without cleaning the third. It could use the function `list_files` so it deletes the same things that would have been read by another command (like load_messages).

Reply:
new commit now calls `list_files` first, then deletes each individual files.